### DISCUSSION TITLE
up主,配置讲的还不够.为什么这么说呢?

### DISCUSSION THREAD

**Discussion by xingzuhui** [BYJRK/DotNet-Discussions#212] [WPF 开发]
因为:
1.runtime还有好多配置方法啊..net配置不看不知道,一看吓一跳.
关于配置的名称空间和类居然有这么多.如下:
![捕获](https://github.com/BYJRK/DotNet-Discussions/assets/33748832/29643efe-5e3c-4542-a3dc-715a2ea216bf)
![捕获2](https://github.com/BYJRK/DotNet-Discussions/assets/33748832/d9cda256-596d-4fba-80f8-2284cf65d9d0)

**Comment by BYJRK** [comment]
回头再讲 appsettings.json 相关的几个
